src/FSharp.Stats/Integration/Differentiation.fs

```diff
@@ -22,3 +26,56 @@ module Differentiation =
         let b  = (x1 - x0 - a*t1*t1)/t1
         (2.*a*t) + b
 
+    ///correcture of the Array.iterUntili function; here no boundarys are set to the lower and upper end
```

Comment:
The description should be standalone. I suggest to copy the original description and adapt it